clubs_guides/urls.py

```diff
@@ -1,3 +1,18 @@
 from django.conf.urls import url
 
-# Create you url mappings here.
+from clubs_guides.views import (
+    ClubsGuidesListView,
+    ClubsGuideView,
+    CategoryListView,
+)
+
+
+urlpatterns = [
```

Comment:
It will be valuable to add comment docs that explain which routes *actually* exist with these patterns.